server/models/Notification.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  type: {
+    type: String,
+    required: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  read: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
+  link: {
+    type: String,
+    required: false,
+  },
+  user: {
+    type: mongoose.Types.ObjectId,
+    ref: "user",
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+});
+
+module.exports = {
+  Notification: mongoose.model("notification", notificationSchema),
+  notificationSchema,
+};
```

Comment:
This looks good for the Schema. I can't think of anything else I want to track for notifications at this time.